aroquev/src/main/webapp/style.css

```diff
@@ -1,3 +1,36 @@
+body {
+  font-family: 'Noto Sans JP', sans-serif;
+}
+
+/* For the header */
+/* Add a black background color to the top navigation */
+.topnav {
+  background-color: #333;
+  overflow: hidden;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: left;
```

Comment:
I don't know if you really need to use `float` for this layout. You can get the same thing if you give the parent div `display: flex` and `flex-wrap: wrap` if you still want a nav link to flow down to the next line on small viewports.

Reply:
So I just added those properties to the .topnav rule?